src/components/index.js

```diff
@@ -0,0 +1,5 @@
+export { default as Filters } from "./Filters/Filters";
```

Comment:
you should do named import and export.

Reply:
Thanks Saurabh for reviewing, I have updated it now with name import and export.